archibald/cogs/alias.py

```diff
@@ -0,0 +1,57 @@
+import discord
+import json
+from discord import option
+from discord.ext import commands
+
+class link_alias(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.link_map = json.loads(open("archibald/cogs/links.json").read())['links'];
+        self.add_alias_whitelist = [
```

Comment:
Let's use [cog checks](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.Cog.cog_check) to check for member roles to include admin instead of hardcoding ids